app/src/main/java/com/codepath/apps/critterfinder/fragments/SwipeablePetsFragment.java

```diff
@@ -121,6 +121,7 @@ public void onAdapterAboutToEmpty(int itemsInAdapter) {
     @Override
     public void onScroll(float scrollProgressPercent) {
         View view = mCardContainer.getSelectedView();
+        view.findViewById(R.id.swipe_card_background).setAlpha(0); // hide the background frame on the card
```

Comment:
This is a hacky way to hide the framing for the current card when you are dragging it around so you don't see the frame. If you take one card off the deck you want the rest of the deck to look like it is stationary while that one card moves around.